src/removeDuplicates.js

```diff
@@ -0,0 +1,5 @@
+// JavaScript function to remove duplicate elements from an array
+
```

Comment:
I think there is no need for adding this comment on the top of the function since the name and the codes are self explanatory  but overall Good Job 🎉